simply/scenario.py

```diff
@@ -171,16 +250,19 @@ def load(dirpath, data_format):
 
 def create_random(num_nodes, num_actors, weight_factor):
     pn = power_network.create_random(num_nodes)
-    actors = [actor.create_random("H" + str(i)) for i in range(num_actors)]
-
     # Add actor nodes at random position (leaf node) in the network
     # One network node can contain several actors (using random.choices method)
```

Comment:
move comments to corresponding code also down

Reply:
Added comment to every actor_nodes generation through random.sample and made the distinction to power_network.create_random() more clear, which only generates the nodes and does not interact with actors yet